pkg/Library/NodeGraph/NodesConnector.js

```diff
@@ -226,12 +252,12 @@ constructInspectParticle(key, spec, storeName, inspectorData) {
   return particle;
 },
 
-buildParticleSpecs(nodeType, node) {
+buildParticleSpecs(nodeType, node, globalStores) {
   const specs = {};
   const names = this.getParticleNames(nodeType) || [];
   for (const particleName of names) {
-    //specs[`${node.key}${this.nameDelim}${particleName}`] = this.buildParticleSpec(nodeType, node, particleName);
-    specs[`${node.key}`] = this.buildParticleSpec(nodeType, node, particleName);
+    specs[`${node.key}${particleName}`] =
+        this.buildParticleSpec(nodeType, node, nodeType[particleName], `main#runner`, globalStores);
```

Comment:
`main#runner`?

Reply:
it's a default value, it used to be in `resolveContainer` before 🤷